test/distributed/_composable/fsdp/test_fully_shard_training.py

```diff
@@ -536,7 +514,7 @@ def _test_train_parity_with_activation_checkpointing(
                 n_heads=4,
                 vocab_size=vocab_size,
                 max_seq_len=64,
-                dropout_p=0.1,
+                dropout_p=0,
```

Comment:
Is this because `torch.compile` does not have same randomness as eager?
Should we just disable it for compile path then?